internal/utils/envvar.go

```diff
@@ -59,23 +59,22 @@ func GetProviderEnvVars(store *druidv1alpha1.StoreSpec) ([]corev1.EnvVar, error)
 		return nil, fmt.Errorf("storage provider is not recognized while fetching secrets from environment variable")
 	}
 
-	const credentialsMountPath = "/var/etcd-backup"
 	switch provider {
 	case S3:
-		envVars = append(envVars, GetEnvVarFromValue(common.EnvAWSApplicationCredentials, credentialsMountPath))
+		envVars = append(envVars, GetEnvVarFromValue(common.EnvAWSApplicationCredentials, common.VolumeMountPathNonGCSProviderBackupSecret))
```

Comment:
Just for my understanding, the methods GetEnvVarFromFieldPath & GetProviderEnvVars & GetEnvVarFromSecret are used only within this util, so not sure why they are declared as public.

Reply:
Nice catch. Will make the change.